src/openInBrowser.js

```diff
@@ -0,0 +1,14 @@
+const opn = require('opn');
+
+const openInBrowser = (url, browser) => {
+  try {
+    const options = typeof browser === 'string' ? {app: browser} : undefined;
```

Comment:
More readable alternative:
```javascript
const options = {app: browser || ''};
```

Reply:
the problem is when we pass `--open` without any browser, the `command.open` passed here as `browser` takes on value as `true`. therefore i did a simple type check to see if it is a string or boolean.
This was because passing `{app: true}` to open , doesn't do anything at all.